Intro page content update

markgreenburg opened the pull request:
Updates content per latest copy in attached ticket.
DOES NOT touch form start controls, so the buttons aren't dynamic and don't do anything. This will be addressed in a separate PR.

Screenshot 1:
![screen shot 2018-08-03 at 2 00 14 pm](https://user-images.githubusercontent.com/24251447/43660791-9f46800c-9725-11e8-9ed4-81db22fa8ae2.png)


Screenshot 2:
![screen shot 2018-08-03 at 2 00 26 pm](https://user-images.githubusercontent.com/24251447/43660803-a4d0bfe2-9725-11e8-842f-b1a13c14f26a.png)




Screenshot 3:
![screen shot 2018-08-02 at 5 58 51 pm](https://user-images.githubusercontent.com/24251447/43615563-de1a9708-967d-11e8-8404-300f62443dc6.png)

markgreenburg committed:
Initial content commit

markgreenburg committed:
Fix form title

markgreenburg committed:
Add comment about SiP Intro

cvalarida commented on src/applications/disability-benefits/all-claims/sass/disability-benefits.scss:
```diff
@@ -12,3 +12,11 @@
   margin-bottom: 2em;
 }
+
+button {
```
I think this is probably too broad a rule to put on the whole form. Is this a temporary fix?

markgreenburg replied:
Yeah, this will need to change, but might be a good idea to make this more explicit.

cvalarida commented on src/applications/disability-benefits/all-claims/sass/disability-benefits.scss:
```diff
@@ -12,3 +12,11 @@
+  margin-top: 1em;
+}
+
+.list-four {
```
Alternatively, `.process-step:last-child` should do the trick as well (and scale better, though that's not strictly speaking necessary here).

markgreenburg replied:
Oh hey, that's a thing!

cvalarida reviewed (approved):
My only concern is applying a too-broad rule to all the buttons in the form. Other than that, it looks good to me if it looks good to @peggygannon.

markgreenburg committed:
Update styling

markgreenburg committed:
Add a period

peggygannon reviewed (changes requested):
@markgreenburg Hi, Mark, see below for some copy tweak to the intro page: 
close up the space above and below the "and" in the bullet list. So it should look like this: "and" is run in on that first bullet line and bolded. 
**You'll be submitting an original disability claim, if both of these are true:**
*You’re a Veteran or Servicemember who’s between 180 and 90 days from ending your military service, **and**
*This is the first time you’re filing a disability claim

this line should be a link:
 [You can order service medical records through the National Archives.](https://www.archives.gov/veterans/military-service-records)

Please update the copy in the Disability ratings blue box to this (adding in info about an exam): 
For each disability claim, we assign a severity rating from 0% to 100%. We base this rating on the supporting documents you turn in with your claim. In some cases we may also ask you to have an exam to help us rate your disability.

markgreenburg commented:
Thanks @peggygannon - here are a few updated screenshots for your review:

Updated ratings box:
![screen shot 2018-08-03 at 5 21 58 pm](https://user-images.githubusercontent.com/24251447/43668297-c9370b2c-9741-11e8-8973-17660ba9c0aa.png)

----------
Updated header and list item formatting:
![screen shot 2018-08-03 at 5 21 51 pm](https://user-images.githubusercontent.com/24251447/43668298-c94cfdba-9741-11e8-9056-a072c8d65432.png)

----------
Link to archives:
![screen shot 2018-08-03 at 5 23 50 pm](https://user-images.githubusercontent.com/24251447/43668338-04345f86-9742-11e8-9a9f-b45065034205.png)

markgreenburg committed:
Update formatting, content revisions
